local_units/models.py

```diff
@@ -82,7 +335,9 @@ class LocalUnit(models.Model):
     )
     draft = models.BooleanField(default=False, verbose_name=_('Draft'))
     validated = models.BooleanField(default=False, verbose_name=_('Validated'))
-    is_public = models.BooleanField(default=False, verbose_name=_('Is public?'))
+    visibility = models.IntegerField(
+        choices=VisibilityChoices.choices, verbose_name=_('visibility'),
+        default=2)  # 2:IFRC
```

Comment:
We also need to verify this with new data excel file.

Reply:
What do you mean on verify?
(And which one is the "new" data excel file?)